src/libcollections/slice.rs

```diff
@@ -437,8 +437,8 @@ impl<T> [T] {
     /// The caller must ensure that the slice outlives the pointer this
     /// function returns, or else it will end up pointing to garbage.
     ///
-    /// Modifying the slice may cause its buffer to be reallocated, which
-    /// would also make any pointers to it invalid.
+    /// Modifying the container this slice references may cause its buffer
```

Comment:
Yeah, I was hoping to have it be a generalized term, even though it's not "standard" per se. In this context, I'd call an array a "container", given that it's what holds the ownership of its items. Could go either way, what with the heap-allocated variable-length things being in "lib*collections*". `:/`